### DISCUSSION TITLE
Trying to get only NPC actors to be labeled "Dead" at both 0 HP & negative HP.

### DISCUSSION THREAD

**Discussion by Oryan77** [mclemente/healthEstimate#231] [Q&A]
I started using Health Estimate and my players are loving it!

After seeing some of the things that can be done by adding new tables and JS Rules, I'm wondering if this is a possibility, and if so, how would I set it up.
 
I currently have Monk's Combat Details set to mark NPCs as "Dead" at Zero HP. So Health Estimate will identify an NPC actor as "Dead", but only if it has exactly 0 HP. 

I have Health Estimate set to label a "Dying" token as "Dying" (-1 thru -9), which is meant to be for PC actors that don't die until -10 HP. If damage is auto-calculated and is enough damage to push an NPC's health beyond 0 (into the negatives), my system (3.5e) marks the NPC as "dying, unconscious, & helpless".  So it skips the 0 = Dead label.

I understand this might be impossible via Health Estimate, but is there a code that could be used via a JS Rule that could disable the "dying, unconscious, & helpless" status effects and replace them with the Dead status effect (and Dead Health Estimate label) if an NPC actor is within -1 to -9 HP?

I apologize if this is a ridiculous request. I'm not all that familiar with what these JS Rule codes can do.

**Comment by mclemente** [comment]
Code to remove the status effects through Health Estimate is out of question.

I've pushed an update that should fix the detection, though. So the NPC setting should mark them as Dead if they have the Skull or the Dead status effect.
